scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -688,12 +689,48 @@ class FormatWriter(formatOps: FormatOps) {
         private def formatListTerm(
             itemIndent: String
         )(item: Scaladoc.ListItem): Unit = {
-          val words = item.text.parts.iterator.map(_.syntax)
+          val words = item.text.part.iterator.map(_.syntax)
           iterWords(words, appendBreak, itemIndent.length - 1, itemIndent)
           appendBreak()
           item.nested.foreach(formatListBlock(itemIndent))
         }
 
+        private def formatTable(table: Scaladoc.Table): Unit = {
+          val rows = table.row.view :+ table.header
+          val align = table.align
+          val maxCols = rows.map(_.col.length).max
+          val colsRange = 0 until maxCols
+          val maxLengths = colsRange.map { x =>
+            rows.filter(_.col.length > x).map(_.col(x).length).max
```

Comment:
```suggestion
            rows.collect { case r if r.col.length > x => r.col(x).length }.max
```

One list traverse instead of two